build.sh

```diff
@@ -29,8 +34,7 @@ podman save --format oci-archive -o "${OUTDIR}/${DISK_IMAGE_NAME}" "${FULL_IMAGE
 
 echo "Transforming OCI image into disk image"
 SRCDIR="${TMT_TREE:-..}"
-pushd $OUTDIR && sudo sh $SRCDIR/build-podman-machine-os-disks/build-podman-machine-os-disks.sh "${PWD}/${DISK_IMAGE_NAME}"
-
+pushd $OUTDIR && sudo sh $SRCDIR/custom-coreos-disk-images/custom-coreos-disk-images.sh --ociarchive "${PWD}/${DISK_IMAGE_NAME}" --platforms applehv,hyperv,qemu
```

Comment:
this needs to add --osname fedora-coreos
This is causing the x86 test failure as it defaults to rh coreos which uses a bigger image size

Anyway on cirrus the aarch64 build failed with the same out of space errors so it doesn't seem env related.